File: java/dayOne/levelThree/TaskThree.java

import java.util.Arrays;

public class TaskThree {

    public static int [] mergingTwoArrays(int [][] numbers){

        int rowLength = 0; 
            
        for(int [] row : numbers){
            
            rowLength = rowLength + row.length; 

            }
        
        
            int [] newArray = new int [rowLength];
        
            int index = 0;
            
        for(int [] row : numbers){
        
            for(int number : row){
            
                newArray[index] = number;
                 
                 index++;
            }
 
        }
   
        for(int row = 0; row < newArray.length; row++){
            for(int count = row + 1; count < newArray.length; count++){
                
                int temporary = newArray[row];
                if(newArray[count] < newArray[row]){
                   newArray[row] = newArray[count];
                   newArray[count] = temporary; 
                   
                }
            
            
            }
        
        }    
   
        return newArray;
                                  
    }

}

File: python/day_one/level_three/task_three.py

def merging_two_arrays(numbers): 

    new_list = []
  
    for row in numbers:
        
        for number in row:
        
            new_list.append(number)
            
    for count_one in range(len(new_list)):

        for count_two in range(count_one + 1, len(new_list)):

            if (new_list[count_one] > new_list[count_two]):
            
                temporary = new_list[count_one]
                new_list[count_one] = new_list[count_two]
                new_list[count_two] = temporary
            
            
            
    return new_list
